test/run_test.py

```diff
@@ -1182,14 +1180,16 @@ def parse_args():
             or TEST_WITH_ASAN
             or (
                 strtobool(os.environ.get("TD_DISTRIBUTED", "False"))
-                and os.getenv("TEST_CONFIG") == "distributed"
+                and TEST_CONFIG == "distributed"
                 and TEST_CUDA
             )
             or (IS_WINDOWS and not TEST_CUDA)
+            or TEST_CONFIG == "nogpu_AVX512"
+            or TEST_CONFIG == "nogpu_NO_AVX2"
         )
         and os.getenv("BRANCH", "") != "main"
```

Comment:
I wonder if you will get a merge conflict with the change in https://github.com/pytorch/pytorch/pull/125485 :)